server-spi/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -162,6 +162,12 @@ public static void importRealm(KeycloakSession session, RealmRepresentation rep,
         else newRealm.setSsoSessionIdleTimeout(1800);
         if (rep.getSsoSessionMaxLifespan() != null) newRealm.setSsoSessionMaxLifespan(rep.getSsoSessionMaxLifespan());
         else newRealm.setSsoSessionMaxLifespan(36000);
+
+        if (rep.getSsoSessionIdleTimeoutRememberMe() != null) newRealm.setSsoSessionIdleTimeoutRememberMe(rep.getSsoSessionIdleTimeoutRememberMe());
+        else newRealm.setSsoSessionIdleTimeoutRememberMe(36000);
```

Comment:
Same as above. It should not be set by default and use non-remember me values if not specified explicitly.